app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,105 @@
+class OrdersController < ApplicationController
+  def index
+    # @orders = Order.find_by(session_id: session[:session_id])
+    if session[:user_id]
+      @user = User.find_by(id: session[:user_id])
+      @products = Product.where(user_id: session[:user_id])
+      @order_items = OrderItem.where(product_id: @user.products)
+      if !@order_items.empty?
+        @order_items_paid = []
+        @order_items_complete = []
+        @order_items.each do |item|
+          (@order_items_paid << item) if item.order.status == "paid"
+          (@order_items_complete << item) if item.order.status == "complete"
+        end
+      #NEEDS ADDITIONAL CLAUSES TO IDENTIFY ORDER FROM ORDER ITEMS
```

Comment:
Is this comment a TODO/note to self?  It's a little confusing.  If its about logic you'd like to add in the future, make sure to put that in your read me, but take it out here.  When I see something like 'needs' in a comment - my first thought is - "is the developer trying to warn me about a datatype or argument I need to implement the code properly, and if so then why isn't that just evident in the readability about the code?"